### ISSUE TITLE
`tryInverted` does not work with `readBarcodeImagePathString()`

### ISSUE THREAD

**Issue by kaboc** [khoren93/flutter_zxing#72]
Inverted QR codes that can be scanned with `ReaderWidget` cannot be scanned with `readBarcodeImagePathString()`.

```dart
final code = await zx.readBarcodeImagePathString(
  'image path',
  params: Params(
    tryHarder: true,
    tryInverted: true,
  ),
);
```

Commenting out `tryHarder: true` did not affect the result.

**Comment by kaboc** [comment]
It appears that this was fixed at 0.10.0, right? Inverted QR codes were read correctly after I updated my app to use the new version and replaced `Params` with `DecodeParams`.

**Comment by khoren93** [comment]
Thank you for your message and for confirming that the issue was fixed.

I'll go ahead and close this issue. If you have any further issues or questions, please don't hesitate to reach out.
